### DISCUSSION TITLE
アドベントカレンダーの「購読する」システムへの改善要望

### DISCUSSION THREAD

**Discussion by tsubolabo** [increments/qiita-discussions#598] [要望・アイデア・不具合]
要望は２つです。
+ 無料で閲覧が可能なのに「購読」という言葉を使うと混乱を招く。
+ 参加記事の公開通知が「通知一覧」へ入るのは困る。

「購読する」ボタンを見て、今年から閲覧が有料になったかのと勘違いし、クリックをためらいました。しかし、閲覧料金はどこにも明示されていません。まさかと思い調べたところ、最近のIT業界では、無料の場合でも「購読」を使う例があることは分かりました。しかし、明らかに誤用です。IT業界を主導しこれらを是正すべき立場にあるQiitaが、自らのページの中で使うべき言葉ではないと思います。

私にとっての「通知一覧」は、自分の記事に「いいね」が付いたことを知らせてくれる「お祝いの鐘」です。一覧を見返しながら、過去の努力の成果を確認し「自己満足する場」でもあります。いくら興味を持っているカレンダーのテーマであっても、全部の投稿通知がここに詰め込まれるのでは大混乱になり、自分の成果だけを見返す楽しみも減ります。一旦「購読する」に登録してみましたが、上記の問題を感じたため、一件の通知を受けただけで、即座に登録を取り消しました。

「購読する」を「フォローする」に変え、アドベントカレンダーのページの頭に「フォロー中のテーマ」（「マイページ」の「フォロー中のタグ」に相当するもの）の欄を表示して、フォロー登録しているテーマのボタンをクリックすると、そのページにジャンプできるようになれば便利です。現状のシステムでは、投稿予約して「参加」している場合には、その参加ページにジャンプするのは容易ですが、「参加」せずに「閲覧するだけ」の場合にはかなり手間がかかります。

**Comment by tanaka-asami415** [comment]
@tsubolabo 
リクエストありがとうございます！🙏

> 無料で閲覧が可能なのに「購読」という言葉を使うと混乱を招く。

確かにご指摘の通りなので、ご提案いただいた内容の通りになるかはお約束ができないのですが
「有料でない」ことは伝わるようにしたいと思います🙇

> 参加記事の公開通知が「通知一覧」へ入るのは困る。

通知欄やカレンダーへの導線（購読、ご自身で作成分）は同様のご意見もいただいており、来年以降アップデート予定です。

**Reply by P-SiZK** [reply]
@tsubolabo 
大変お待たせいたしました :bow:

ご連絡いただいたアドベントカレンダーに関するご要望につきまして、改善を行いましたのでご報告いたします。

まず、「購読」という表現を「フォロー」に変更いたしました。

また、記事公開時の通知に関しましては、通知設定の「フォロー中のアドベントカレンダー」でオンオフを切り替えられるようになりました。
詳細につきましては、下記のリリースノートをご覧ください。
https://qiita.com/release-notes/2025-10-15

さらに、フォロー中のカレンダーの記事をより簡単にご覧いただけるよう、[カレンダーフィードページ](https://qiita.com/advent-calendar/2024/article-feed)にて、カレンダーごとの絞り込み機能を追加いたしました。
これにより、フォロー中のカレンダーの記事が探しやすくなっておりますので、ぜひご活用いただけますと幸いです。

**Reply by tsubolabo** [reply]
@P-SiZK 

改善いただきありがとうございます。

しかし、これ以外にも「購読」を誤用している箇所はあるようです。私が気づいているのは下記の部分です。

Qiitaの各投稿記事のページの三点リーダーをクリックしたときに表示される「記事を購読」「記事の購読を解除」の中の「購読」も意味が分かりません。

普通は「記事を購読」となっていますが、自分がコメントを付けた記事では「記事の購読を解除」と表示され「購読」を申し込んだことにされているようです。しかし、料金を請求される訳ではありません。

**Reply by P-SiZK** [reply]
@tsubolabo 
ご確認ありがとうございます！
記事に対して「フォロー」という表現は少し適切でないという考えに加え、インターネット上のサービスでは無料でも「購読」という表現も一般的であることも考慮し、一旦カレンダー以外は「購読」のままとさせていただいております。

今後の表現については、Discussionsで皆さまのご意見もお伺いできればと思います。

**Reply by tsubolabo** [reply]
@P-SiZK 

> インターネット上のサービスでは無料でも「購読」という表現も一般的であることも考慮し

**それがおかしいと言っているのです**。誤用している例が多いというだけで、全然一般的ではありません。以前にも書いているように、**Qiita はこれを進んで是正すべきです**。

最近は詐欺メールが横行しています。いつ何時、この誤用を利用した詐欺が多発することになるのか心配でなりません。怒りに手が震えてマウス操作も儘なりません。

**Reply by tsubolabo** [reply]
@P-SiZK 

> 記事に対して「フォロー」という表現は少し適切でない

適切でないと言われても、ここでの「購読」は何を意味しているのかの定義も曖昧なまま、適切な言い換えなど思い浮かぶはずもありません。失礼ながら、単に以前からの習わしというだけで、ご自分でも十分にはお分かりになっていないのではありませんか？

「記事を購読」：
これはどんな意味ですか？　このボタンは怪しげでクリックしたこともなく、この先に何があるのか全然わかりません。クリックしたら何がどうなるのですか？　ここの貴コメントで「無料」ということまでは分かりましたが、その先にどんなメリット、デメリットが待っているですか？

「記事の購読を解除」：
上記が分からないので、こちらも分かるはずがありません。しかし、自分がコメントを書いた記事ではこの表示が現れるところを見ると、コメントを書くと、「記事を購読」をクリックしたこともないのに、勝手に購読を申し込んだことになっているようです。一体、「購読」とは何なんですか？

**Reply by shiracamus** [reply]
外野から失礼します。
Subscribeの和訳が購読ですよね。
このGitHubのQiitaDiscuttionのページの右側にもNotifications の Subscrive/Unsubscribeがあって、和訳すると「通知を講読する」ですね。

ユーザ：フォロー
タグ：フォロー
カレンダー（テーマ）：フォロー
記事：購読（コメントすると自動的に講読）

フォローと講読の明確な違いがないのであれば、フォローで統一するのが混乱がなくて良さそうですね。

**Reply by wany-oh** [reply]
記事に対して「フォロー」という表現を使うことに特に疑問を持ちませんが、それをいうなら記事をフォローよりタグをフォローのほうが違和感ありますね。「ウォッチ」とかが良いかもしれませんね。

**Reply by tanaka-asami415** [reply]
@tsubolabo @shiracamus @wany-oh 

コメントありがとうございます！
運営としては記事の場合は「フォロー」という表現に違和感を持たれる方が多いかもしれないと考えていたのですが、実際にはそうでもなさそうですね。

将来的には表現の見直しも検討できればと思いますが、対象によっては「フォロー」以外の表現が適している場合もありそうなため、「フォロー」に統一するかは現時点では未定です。

また、tsubolaboさんからいただいているリクエストの中では、数式関連の内容を優先して対応したいと考えております。
そのため、本件についてはすぐの対応が難しい点、ご了承いただけますと幸いです🙇

お待ちいただいている間に、「こんな表現もありそう」といったアイデアがありましたら、ぜひお聞かせください。

**Reply by tsubolabo** [reply]
@tanaka-asami415 

> お待ちいただいている間に、「こんな表現もありそう」といったアイデアがありましたら、ぜひお聞かせください。

私が本当に訴えたいことは、『無料のものに「購読」という言葉を使わないで欲しい』ということだけなので、それ以上のことを求めると焦点がぼやけてしまいます。しかし、折角のお誘いなので少々申し上げておきたいと思います。

===========

言い換えのための適語を見つけようとする以前に、現在の「購読」という用語を、Qiita さんがどんなニュアンスで使われているのかが分かりません。そこで、伝えようとしているであろう具体的な意味を勝手に想像して、文末にリストアップしてみました。このリストを参照しながら、色々と考えてみます。

「購読」の元々の意味には、「有料で」「定期的に」（刊行物などを）「入手する」という３つの要素が含まれていると思います。書店でたまたま見つけた単行本を購入して読んでも、購読という言葉はあまり使いません（「借りたのではなく買ったんだ」と特に強調したいときには使う人も居るかもしれません）。普通は、「定期的」という意味も含まれていることは間違いありません。となると、「無料で購読」の意味として残るのは、「定期的に」「入手する」となります。

これを考えると、アドベントカレンダで使われていた「購読」は、「無料」という但し書きを強引に付けた場合、かなり適切な使い方だったことになります。「購読」を申し込んだテーマに新規の投稿があったことを、毎日「定期的」に知らせてくれて、その記事を「入手」しに行けるからです。

しかし、記事ページの三点リーダーからの「購読」は、上記の解釈では全く辻褄が合いません。改定もされない同一の記事なのに、その存在を毎日とか毎月とか定期的に報告されても何のメリットもありません。自分が気になる記事であれば、別途用意されている「いいね」や「ストック」の機能を使って便利にフォローすることができます。

ところで、上記において、コメントを付けた記事は自動的に「購読」したことにされ、返信コメントが付いたときには通知が来るようになるところを見ると、「返信の通知を希望する」の意味で使われているのではないかと思われます。相手の都合次第で行われる返信は「定期的」ではありません。こうなると、「購読」の本来の意味の中の「入手する」という要素しか利用されていないことになります。

これらを考え合わせると、『「フォロー」という表現は少し適切でない』どころか、無料という注記をつけたとしても、今までの「購読する」という表現の方がもっと不適切であるということになります。

===========

「購読」という表現のままで問題ないと思っている方々の理由として、次のようなことがありそうです。

> 「購読」は subscribe の和訳である。そして、世の中には無料の subscription も存在する。ゆえに、「購読」という言葉でも「無料」と解釈することは妥当である。

しかし、私の行動範囲が狭いためか、無料の subscription というものは見たことがありません。本当は買い切りのソフトが欲しいのに、subscription 形式のソフトがどんどん増えています。非常に忌々しい思いで広告を見ているのですが、無料のものは見たことがありません。上記の論理で「無料でも購読」を説かれても納得できません。

私の電子辞書では、購読や subscribe は次のように説明されています。上で述べた「購読」に対する私の解釈は、これらを総合的に判断した結果です。

> 広辞苑第六版
書籍・新聞・雑誌などを買って読むこと。

> 明鏡国語辞典第二版
新聞・雑誌などを買って読むこと。
【表現】「購」は、買うの意、本来は代金の発生しないものには言わないが、近年「メールマガジンを購読する」など、単に<定期的>に読むの意でもいう。

> ジーニアス英和辞典第５版
\[新聞・雑誌などを\](予約)定期購読する。\[オンラインのニューズレターなどを\]購読する。\[有料サービスに\]加入する。\[講演の\](シリーズの)チケットを買う。

広辞苑や明鏡には「定期的」であることは明記されていませんが、どう見ても言葉足らずです。一般に世間で使用されるときには、明らかに「定期的」のニュアンスは含まれています。

明鏡の【表現】部は、辞書としての社会的責任を放棄した記述です。誤用を正すべきところを、誤用を煽るような記述になっています。由々しき事です。無視すべきです。（「忘備」の項では誤用であることを明示し、「備忘」を正として社会的責任を果たしているのに）

ジーニアスには、「有料」であり「定期的」でもあることが明示されています。

===========

「購読」の意味として想像した文言のリスト

実際の言い換えでは、これを短縮表現しなければならないし、好みによってカタカナ語の方を望む人も居るだろうし、リストの中の複数の項目を一体にした表現が必要になることもあろうし、と大変ですが、宜しくお願いします。

短縮形で曖昧さが気になる表現のときには、そのボタンをマウスオーバーしたときに、さらに詳しい説明が現れるようになれば言うことなしですが、Qiita さんのメンテナンスが大変になるでしょうから、贅沢な我儘かもしれません。

【リスト】
無料で配信を受ける。
返信の通知を希望する
変更・改定の通知を希望する
変化を検知する。
動向を注視する。
成り行きを注視する。
追っかけになる。
追跡する。
辿る。
目印をつける。
お気に入り。
マークする。
見守る。
ちょっと気になる。

===========

追伸（2025-10-25）：

Qiita での話題としては不適当なものかもしれませんが、「言葉の定義と使い方をはっきりしてくれ」という他の例として…。

最近、熊による人的被害が多発しており、新聞には「捕獲」や「駆除」という単語が頻繁に見られます。素直な解釈として、「捕獲」は「生け捕り」（多くはその後に山へ帰す）、「駆除」は「殺処分」と思っていましたが、どうも、そうでもなさそうです。

「駆除」とはあっても、実際には殺処分まではしていないのでは？と感じさせる記事があります。「熊さんかわいそう」というクレームを躱すための対策かもしれませんが、曖昧な表現は困ります。

辞書によると、

■　広辞苑
捕獲：　とらえること。いけどること。とりおさえること。
駆除：　追い払い、取りのけること。

■　明鏡
捕獲：　動物などを捕らえること。いけどること。
駆除：　追い払って、とりのぞくこと。

驚いたことに、どちらの辞書にも、「駆除」には「殺処分」の意味は含まれていません。辞書もあてにならないものです。「購読」から「定期的」の意味が漏れていたように…。

**Comment by tsubolabo** [comment]
@P-SiZK 

> さらに、フォロー中のカレンダーの記事をより簡単にご覧いただけるよう、カレンダーフィードページにて、カレンダーごとの絞り込み機能を追加いたしました。

11/17 現在、お気に入りのテーマをフォローしているのに、「カレンダーフィード」をクリックしても次のようなコメントしか現れません。

> カレンダーフィード
> まだ表示できる記事がありません
> フォロー中のカレンダーに記事が投稿されていないか、まだカレンダーをフォローしていないため、記事は表示されません。

しかし、当該カレンダーを探して開いてみると、既に複数項目の予約記事が存在してます。12月に入る前でも、予約状況の確認のため、「カレンダーフィード」のクリックでフォロー中のカレンダーが表示されるようにして欲しいものです。そのテーマに参加するかどうかは、このカレンダーの内容を確認してから決めることになります。

追伸：
上記のカレンダーフィードのコメント枠の下端にある「カレンダーを探す」をクリックし、検索ワードを入力しても、去年の内容しか表示されません。

**Reply by shiracamus** [reply]
アドベントカレンダーは「限定共有記事」にしておいて投稿予約できるので、リンクを知ってる人だけが閲覧できる状態かと思います。
私は、予約だけして公開までに記事をアップデートしてたりするので、公開前に未完成な記事を読まれるのは嬉しくないです。
12月になってからアドベントカレンダーが開かれるのを楽しみにしてる人にとっても、未完成な予約記事をカレンダフィードに表示されるのは嬉しくないかと思います。

**Reply by tsubolabo** [reply]
@shiracamus
> 予約記事をカレンダフィードに表示するのは反対じゃないでしょうか。

今回の P-SiZK さんのコメントを見るまでは、今までカレンダフィードというものを利用したことがなかったので、私の見込み違いがあったのかもしれません。

私が想定し期待していたのは、フォローしたテーマについては、カレンダフィードをクリックするだけで、ワンタッチでカレンダーの内容（個々の予約記事の内容ではなく）を見に行けるということでした。

記事自体の内容については、予約日が来るまでは非公開であることは十分に承知しております。

**Reply by P-SiZK** [reply]
@tsubolabo 
> 12月に入る前でも、予約状況の確認のため、「カレンダーフィード」のクリックでフォロー中のカレンダーが表示されるようにして欲しいものです。

[カレンダーフィード](https://qiita.com/advent-calendar/2025/article-feed) は、仕様上、12月1日以降に公開される記事が表示されるページとなっております。
12月以前にフォロー中のカレンダーの状況をご確認いただく場合は、お手数ですが [フォロー中のカレンダー](https://qiita.com/advent-calendar/2025/feed) のページから、各カレンダーの詳細ページへアクセスのうえ、ご確認ください :pray: 

> 上記のカレンダーフィードのコメント枠の下端にある「カレンダーを探す」をクリックし、検索ワードを入力しても、去年の内容しか表示されません。

ご報告ありがとうございます！

[以前のコメント](https://github.com/increments/qiita-discussions/discussions/598#discussioncomment-14695152)時点では、今年のカレンダーページがまだ公開前であったため、2024年のページをご案内しておりました。もしかすると、その影響かもしれません。
今年のカレンダーフィードページ (https://qiita.com/advent-calendar/2025/article-feed) や、Qiita Advent Calendar 2025トップページ (https://qiita.com/advent-calendar/2025) をご利用いただくと、今年のカレンダーがご確認いただけるかと思います。

もし今年のページでも再現するようでしたら、お手数ですが、具体的な手順やご利用の環境 (OS・ブラウザなど) をお知らせいただけますでしょうか？

**Reply by tsubolabo** [reply]
@P-SiZK

> 12月以前にフォロー中のカレンダーの状況をご確認いただく場合は、お手数ですが フォロー中のカレンダーのページから、各カレンダーの詳細ページへアクセスのうえ、ご確認ください 🙏

「カレンダー一覧」のページのメニューに「★フォロー中のカレンダー」があることに気づきませんでした。失礼しました。

> 以前のコメント時点では、今年のカレンダーページがまだ公開前であったため、2024年のページをご案内しておりました。もしかすると、その影響かもしれません。

確認したのは今日だったのですが、現時点ではどんなルートから入っても不具合は再現できず、今年の内容が正しく表示されます。私の勘違いだったのでしょうか。

**Reply by P-SiZK** [reply]
@tsubolabo 
ご確認ありがとうございます！
検索の件、現在は正しく表示されているようでよかったです。

もしまた何か不具合などがございましたら、Discussion までご連絡ください 🙏
